src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpApplicationController.cs

```diff
@@ -1158,14 +1157,21 @@ public async Task<IActionResult> ConfirmSubmitApplication(SubmitApplicationViewM
             // TODO: Validate all sections are completed (i.e all questions answered)
             // FUTURE WORK: Validate all sections have had requested feedback answered
 
+            var providerRouteTask = _qnaApiClient.GetAnswerByTag(model.ApplicationId, RoatpWorkflowQuestionTags.ProviderRoute);
+            var applicationTask = _apiClient.GetApplication(model.ApplicationId);
+            var allSectionsTask = _qnaApiClient.GetSections(model.ApplicationId);
+            var roatpSequencesTask = _apiClient.GetRoatpSequences();
+            var organisationVerificationStatusTask = _organisationVerificationService.GetOrganisationVerificationStatus(model.ApplicationId);
+            var providerRoutesTask = _roatpApiClient.GetApplicationRoutes();
```

Comment:
Can we rename this to something like roatpApplicationRoutesTask? It's too similar providerRouteTask  and confused me for 5 mins :)